sys/sdb_version.py

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env python
```

Comment:
please `make -C shlr sdb-sync` (i've fixed the string version)

Reply:
no, I don't want meson to depend on some magic stuff that make can do. It is already borderline we get the things from a .mk file, but having meson calling `make` would just defeat the whole purpose of having meson.

Reply:
i probably wasnt clear enough, im bnopt telling you to do anything with make or any magic, or any .mk

i just said that the sdb version in master was 1.4.1, not 1.5.0, so i was just asking you to sdb-sync to get the version file fixed. but now its too late. i'm working on a huge api change in sdb and cleaning a lot of ugly code that it's still there.

meson is just python, it can do MUCH mor emagic than make can do

keep laughing @xarkes . because yes. calling make from meson is stupid, but reading my message twice instead of assuming that i'm stupid and asking is much more complicated. feel free to fix the version string by hand i fyou like